cime_config/config_pes.xml

```diff
@@ -1,81 +1,129 @@
 <?xml version="1.0"?>
 
+<!-- OMIP-type compsets:
+     On betzy: single node (XS) = 128 CPU cores   ("devel" job size)
+               4 nodes = 512 CPU cores            (minimum "normal" size)
+-->
+
 <config_pes>
 
+  <!-- Configurations for the one degree tripolar (tnx1) grid  !-->
+
   <grid name="a%T62.+oi%tnx1v1">
```

Comment:
I am wondering if we need to support the tnx1v1 and tnx1v3 grids  any longer. As far as I know, they were never used for any production runs(?) @matsbn might have an opinion about this.